### ISSUE TITLE
Invalid facet value type ignored

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#482]
### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
```raml
#%RAML 1.0
title: A
types:
  myDate:
    type: string
    facets:
      forrrmat: string
  year:
    type: myDate
    forrrmat: YYYY
  yee:
    type: year
    forrrmat: 1332

```

**Comment by BlockLucas** [comment]
https://www.mulesoft.org/jira/browse/APIMF-1519

**Comment by gutee** [comment]
This is fixed and stated for 4.0.0 release
